src/visions/types/boolean.py

```diff
@@ -11,7 +11,12 @@
 
 
 def to_bool(series: pd.Series) -> pd.Series:
-    return series.astype("Bool" if series.hasnans else bool)
+    if int(pd.__version__.split(".")[0]) >= 1:
+        type_name = "boolean"
+    else:
+        type_name = "Bool"
+
+    return series.astype(type_name if series.hasnans else bool)
```

Comment:
Pull the type_name outside of the function so that it's only checked once. i.e.

```
hasnan_bool_name = 'boolean' if int(pd.__version__.split(".")[0]) >= 1 else "Bool"

def to_bool(series: pd.Series) -> pd.Series:
   return series.astype(hasnan_bool_name if series.hasnans else bool)
```
```